.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
You should add the following line in the root .gitignore file:  
`.idea/`

And you should remove these files, either by `git reset --soft HEAD~1` one by one, and then committing again, or by deleting these files in a second commit. The best would be to simply remove this commit from the log.

Reply:
You mean the file .idea should be remove?